go/mysql/conn.go

```diff
@@ -1451,7 +1451,9 @@ func (c *Conn) parseOKPacket(in []byte) (*PacketOK, error) {
 	if c.Capabilities&uint32(CapabilityClientSessionTrack) == CapabilityClientSessionTrack {
 		// info
 		info, _ := data.readLenEncInfo()
-		packetOK.info = info
```

Comment:
Unrelated to this PR, this same code is duplicated in the `if` and `else`. It can be moved prior to the `if`.